.gitignore

```diff
@@ -11,3 +11,4 @@ src/cryptography/_Cryptography_cffi_*
 *.egg
 .eggs/
 *.py[co]
+setup.cfg
```

Comment:
Ah, oversight. Need it to build on windows with non standard Openssl install (x64), otherwise the compile blows up due to wrong pathes. I'll remove it from this PR.